x-pack/plugins/alerts/server/routes/create.ts

```diff
@@ -38,6 +38,7 @@ export const bodySchema = schema.object({
     }),
     { defaultValue: [] }
   ),
+  notifyWhen: schema.nullable(schema.string({ validate: validateNotifyWhenType })),
```

Comment:
per previous comment about making this property optional vs nullable, if we go that route, we can change this to `schema.maybe()`.